composer/loggers/logger.py

```diff
@@ -127,18 +127,6 @@ def file_artifact(
                 overwrite=overwrite,
             )
 
-    def data_fit(self, data: Dict[str, Any]) -> None:
```

Comment:
Can we keep these around, and throw an error? e.g. `raise NotImplementedError(data_fit is no longer a valid call to the logger API. Please use log_hyperparameters or log_metrics instead)

Reply:
done